lighthouse-cli/cli-flags.js

```diff
@@ -62,6 +62,9 @@ function getFlags(manualArgv) {
           'lighthouse <url> --only-categories=performance,pwa',
           'Only run specific categories.')
 
+      // Accept a file for all of these flags.
+      .config('cli-settings-path')
```

Comment:
can you add a description specifying that other flags will override this one? (e.g. `--throttling-method=provided` on the command line will override `"throttling-method": "devtools"` in the settings file)

Reply:
also, bikeshed time: should we call it something like `cli-flags-path`?

Reply:
also, copy over to readme :)

Reply:
> can you add a description specifying that other flags will override this one?

Yes, test added too! 👍 

> cli-flags-path

WFM!

> also, copy over to readme 

This feels close to an autopopulate script too if only we made changes more frequently 🤔

Reply:
> This feels close to an autopopulate script too if only we made changes more frequently 🤔

definitely, and it's a little silly we have it at all, but I know I've at least found it useful in the past, so....